File: Day 0/0 Count Primes/solution.cs
public int CountPrimes(int n) {
    int count = 0;
    for (int i = 2; i < n; i++) {
        if (IsPrime(i)) count++;
    }
    return count;
}

bool IsPrime(int n) {

    if (n <= 1) 
        return false;

    for (int i = 2; i <= Math.Sqrt(n); i++) {
        if (n % i == 0) 
        return false;
    }

    return true;
}

bool IsPrime(int n) {
    if (n <= 1) return false;

    for (int i = 2; i * i < n  ; i++) {
        if (n % i == 0) 
        return false;
    }
    return true;
}

File: Day 0/0 Count Primes/solution.js
var countPrimes = function(n) {
    let count = 0;
    for (let i = 2; i < n; i++) {
        if (isPrime(i)) count++;
    }
    return count;
};

var isPrime = function(n) {
    if (n <= 1) return false;
    for (let i = 2; i <= Math.sqrt(n); i++) {
        if (n % i == 0) return false;
    }
    return true;
};
